calico/security/encrypt-cluster-pod-traffic.md

```diff
@@ -141,9 +147,6 @@ To install WireGuard for OpenShift v4.8:
 
 #### Enable WireGuard for a cluster
 
-   > **Note**: Nodes that do not support WireGuard will not be secured by WireGuard tunnels, even if traffic running on the node to and from the pods goes to nodes that do support WireGuard.
```

Comment:
Why remove this note?

Reply:
This was the text that Support said was confusing. The content was moved to Before  you begin so doesn't need to be repeated here.